public/sass/components/_scrollbar.scss

```diff
@@ -106,78 +106,78 @@
   opacity: 0.9;
 }
 
-// // Scrollbars
-// //
-//
-// ::-webkit-scrollbar {
-//   width: 8px;
-//   height: 8px;
-// }
-//
-// ::-webkit-scrollbar:hover {
-//   height: 8px;
-// }
-//
-// ::-webkit-scrollbar-button:start:decrement,
-// ::-webkit-scrollbar-button:end:increment {
-//   display: none;
-// }
-// ::-webkit-scrollbar-button:horizontal:decrement {
-//   display: none;
-// }
-// ::-webkit-scrollbar-button:horizontal:increment {
-//   display: none;
-// }
-// ::-webkit-scrollbar-button:vertical:decrement {
-//   display: none;
-// }
-// ::-webkit-scrollbar-button:vertical:increment {
-//   display: none;
-// }
-// ::-webkit-scrollbar-button:horizontal:decrement:active {
-//   background-image: none;
-// }
-// ::-webkit-scrollbar-button:horizontal:increment:active {
-//   background-image: none;
-// }
-// ::-webkit-scrollbar-button:vertical:decrement:active {
-//   background-image: none;
-// }
-// ::-webkit-scrollbar-button:vertical:increment:active {
-//   background-image: none;
-// }
-// ::-webkit-scrollbar-track-piece {
-//   background-color: transparent;
-// }
-//
-// ::-webkit-scrollbar-thumb:vertical {
-//   height: 50px;
-//   background: -webkit-gradient(
-//     linear,
-//     left top,
-//     right top,
-//     color-stop(0%, $scrollbarBackground),
-//     color-stop(100%, $scrollbarBackground2)
-//   );
-//   border: 1px solid $scrollbarBorder;
-//   border-top: 1px solid $scrollbarBorder;
-//   border-left: 1px solid $scrollbarBorder;
-// }
-//
-// ::-webkit-scrollbar-thumb:horizontal {
-//   width: 50px;
-//   background: -webkit-gradient(
-//     linear,
-//     left top,
-//     left bottom,
-//     color-stop(0%, $scrollbarBackground),
-//     color-stop(100%, $scrollbarBackground2)
-//   );
-//   border: 1px solid $scrollbarBorder;
-//   border-top: 1px solid $scrollbarBorder;
-//   border-left: 1px solid $scrollbarBorder;
-// }
+// Scrollbars
 //
+
+::-webkit-scrollbar {
```

Comment:
sadly the custom scrollbar styles breaks native overlay scrollbars, so we need to figure out a way to add this only if there is no native scrollbars, so please remove this from this PR

Reply:
I understand, and it doesn't really have anything to do with the feature so it was probably never appropriate to include anyway.